src/modules/ROOT/pages/changelog/202309/01.json

```diff
@@ -0,0 +1,89 @@
+{
+  "meta": {
+    "releasedOn": "2023.12.01"
+  },
+
+  "added": [
+    {
+      "what": [
+        "the options to clone and copy a page block within a tabbed page block."
+      ],
+      "why": [
+        "to provide an easier way to clone and copy tabbed page blocks."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1527"
+      ]
+    },
+    {
+      "what": [
+        "a dropdown to select the number of items to be displayed per page in a list of resources."
+      ],
+      "why": [
+        "to provide the user with the option to select how many resources can be displayed per page."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1534"
+      ]
+    }
+  ],
+
+  "changed": [
+    {
+      "what": [
+        "the error message to be more verbose when user includes invalid or missing record input required values in a workflow function."
+      ],
+      "why": [
+        "to provide a clearer message on what and where the issue is."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1571"
+      ]
+    }
+  ],
+
+  "fixed": [
+    {
+      "what": [
+        "undefined property error in the developer console after adding a tabs page block without providing a tab in the configuration."
```

Comment:
We don't need to mention console error fixes. Instead, it can be reworded to how it was fixed from their perspective.
For example in this case the tab block now better handles wrong tab configurations.